src/univariate_lp_relaxation.jl

```diff
@@ -24,26 +25,32 @@ function _build_univariate_lp_relaxation!(
     x::JuMP.VariableRef,
     y::JuMP.VariableRef,
     univariate_function_data::UnivariateFunctionData,
-    pre_base_name::AbstractString
+    variable_pre_base_name::AbstractString,
+    constraint_pre_base_name::AbstractString,
+    ::FormulationInfo
 )::FormulationInfo
     vertices = _get_lp_relaxation_vertices(univariate_function_data)
     num_vars = length(vertices)
     formulation_info = FormulationInfo()
 
     # add variables 
-    lambda =
-        formulation_info.variables[:lambda] =
-            @variable(m, [1:num_vars], 
-                lower_bound = 0.0, upper_bound = 1.0, 
-                base_name = pre_base_name * "_lambda")
+    lambda = @variable(m, [1:num_vars], 
+            lower_bound = 0.0, upper_bound = 1.0, 
+            base_name = variable_pre_base_name * "lambda")
```

Comment:
Could a similar `reuse` parameter be specified for this function as in `_build_univariate_milp_relaxation!`, assuming an equivalent partitioning? It seems like these variables could be reused across multiple univariate relaxations.